src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -138,13 +138,31 @@ const statusesToIncludeInQuery = {
   [TaskStatus.Unstarted]: true
 };
 
+const getStatuses = (rawStatuses: string[] | string) => {
+  const statuses = getArray(rawStatuses).filter(
+    status => status in statusesToIncludeInQuery
+  );
+  if (
```

Comment:
we can remove this and apply the fix we came up with earlier where we save the variables for useQuery in a useState to ensure they don't change between renders.

Reply:
you already applied the fix in a previous PR of yours